src/devices_models/devices/AC_branches.jl

```diff
@@ -39,6 +39,20 @@ get_variable_binary(
 ) = false
 
 get_variable_multiplier(_, ::Type{<:PSY.ACBranch}, _) = NaN
+
+function initialize_timeseries_names(
```

Comment:
I don't understand why we have time series here. No model in branches uses time series.